clusters/app.ci/ci-operator/registry-push-credentials-ci-central_secret_template.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+objects:
+- apiVersion: v1
+  kind: Secret
+  metadata:
+    annotations:
+      openshift.io/description: |
+        This secret contains a docker config which has the permissions to pull the images from local registry
+        and push the images to the ci central registry.
+
+    name: registry-push-credentials-ci-central
+    namespace: ${namespace}
+  stringData:
+    .dockerconfigjson: |-
```

Comment:
Can't we use @droslean new feature here?

Reply:
Let me read the code.

Reply:
well this template represents the output right? The new feature will generate this kind of output here either way.

Reply:
Done.

I will remove `secret/ci-central-registry-push-tokens` on all clusters after merge.